webpack/containers/Application/config.js

```diff
@@ -38,11 +38,11 @@ export const links = [
   },
   {
     path: 'module_streams',
-    component: withHeader(ModuleStreams, { title: __('Module Streams') }),
+    component: WithOrganization(withHeader(ModuleStreams, { title: __('Module Streams') })),
   },
   {
     path: 'module_streams/:id([0-9]+)',
-    component: withHeader(ModuleStreamDetails, { title: __('Module Stream Details') }),
+    component: WithOrganization(withHeader(ModuleStreamDetails, { title: __('Module Stream Details') })),
```

Comment:
Do ansible collections need the same treatment?

Reply:
oh quite possible @jjeffers @sajha ^

Reply:
hmm samir stated the same above. Going to check

Reply:
You didn't give your treatment to ansible collections with the last commit.. :smile:

Reply:
Should be fixed for Ansible collections also